flow/shared/postgres/type_conversion.go

```diff
@@ -0,0 +1,144 @@
+package postgres
+
+import (
+	"github.com/jackc/pgx/v5/pgtype"
+	"github.com/lib/pq/oid"
+
+	"github.com/PeerDB-io/peerdb/flow/shared"
+	"github.com/PeerDB-io/peerdb/flow/shared/types"
+)
+
+type UnsupportedError struct {
```

Comment:
Should implement `Error() string` method to fulfill error interface. & then error handler grabbing Message should call Error() instead